src/plugins/data_table/data_table.js

```diff
@@ -279,18 +259,19 @@ var LocalDataTable = (function() {
         // Iterate over the current columns config
         this.columnsConfig().forEach(function(col) {
           if (this.columnVisibility(col.id) || col.bulk) {
+            var node = $("<td>");
+            // If column is a non totals column, draw "Grand totals" on the first one and the rest are empty
             if (this.isNontotalsColumn && this.isNontotalsColumn(col)) {
-              // If column is a non totals column, draw "Grand totals" on the first one and the rest are empty
               if (hasGrandTotalsCell) {
-                $grandTotalsRow.append($("<td>"));
+                $grandTotalsRow.append(node);
               } else {
                 hasGrandTotalsCell = true;
-                var grandTotalsCellText = this.totalsRow.hasUniques ? "Total Gross<br>Total Net" : "Grand Total";
-                $grandTotalsRow.append($("<td>").addClass('.grand-total-title').html(grandTotalsCellText));
+                node.addClass('.grand-total-title');
```

Comment:
Yeah I can remove it. I was hoping to use it in web side, but ended up not. Thanks for catching this